package.json

```
@@ -53,7 +53,7 @@
     "autoprefixer": "^10.4.0",
     "babel-plugin-istanbul": "^6.1.1",
     "babel-plugin-jsx-remove-data-test-id": "^3.0.0",
-    "cypress": "^9.1.0",
+    "cypress": "^9.0.0",
```

```suggestion
    "cypress": "9.0.0",
```

`^` will update it still, need to set it to 9.0.0 explicitly and run `npm i` again.